code/datums/outfits/ert/iac.dm

```diff
@@ -0,0 +1,87 @@
+/datum/outfit/admin/ert/iac
+	name = "IAC Responder"
+
+	uniform = /obj/item/clothing/under/rank/iacjumpsuit
+	suit = /obj/item/clothing/suit/storage/iacvest
+	head = /obj/item/clothing/head/helmet/iachelmet
+	mask = /obj/item/clothing/mask/balaclava/iacmask
+	shoes = /obj/item/clothing/shoes/iac
+	gloves = /obj/item/clothing/gloves/white
+	belt = /obj/item/storage/belt/medical
+	back = /obj/item/storage/backpack/satchel_med
+	accessory = /obj/item/clothing/accessory/storage/white_vest
+	accessory_contents = list(/obj/item/reagent_containers/hypospray/cmo = 1, /obj/item/storage/pill_bottle/dexalin_plus = 1, /obj/item/storage/pill_bottle/tramadol = 1)
+	id = /obj/item/card/id/distress/iac
+
+	l_ear = /obj/item/device/radio/headset/distress
+
+	backpack_contents = list(
+		/obj/item/storage/box/survival = 1,
+		/obj/item/storage/firstaid/regular = 2,
```

Comment:
I'd make this an adv medkit tbh.

Reply:
regular's better than adv with brainmed i think